azplugins/pair.py

```diff
@@ -199,6 +199,66 @@ def process_coeff(self, coeff):
 
         return _hoomd.make_scalar4(epsilon, sigma**3, sigma**6, _hoomd.int_as_scalar(style))
 
+class hertz(hoomd.md.pair.pair):
+    R""" Hertz potential
+
+    Args:
+        r_cut (float): Default cutoff radius (in distance units).
+        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
+        name (str): Name of the force instance.
+
+    :py:class:`hertz` is the Hertz potential
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}{
+        V(r)  = & \varepsilon (1-\frac{r}{r_{\mathrm{cut}}})^{5/2} & r < r_{\mathrm{cut}} \\
+              = & 0 r \ge r_{\mathrm{cut}}
+        \f}
+        \end{eqnarray*}
```

Comment:
I think you still need to change this like so:
```suggestion
    :py:class:`hertz` is the Hertz potential:

    .. math::

        V(r)  &= \varepsilon \left(1-\frac{r}{r_{\rm cut}}\right)^{5/2},& r < r_{\rm cut} \\
              &= 0,& r \ge r_{\mathrm{cut}}
```